### PR TITLE
Make dynamic EGL loading work.

### PR DESCRIPTION
**Pull request by jdm**

### CODE REVIEW

**Comment by jdm** [comment]
These changes makes ANGLE support in Servo work again.

**Comment by jdm** [comment]
@kvark Could you sign off on these changes? The last ANGLE update made it so that libEGL loads libGLESv2, so we need to build both DLLs now to successfully load the functions necessary for an EGL context.

File: gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/Renderer11.cpp [hunk 1]
```diff
@@ -414,7 +414,7 @@ Renderer11DeviceCaps::Renderer11DeviceCaps() = default;
 
 Renderer11::Renderer11(egl::Display *display)
     : RendererD3D(display),
-      mCreateDebugDevice(false),
+      mCreateDebugDevice(true),
```
Comment (kvark) [inline gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/Renderer11.cpp]: Is this intentional? We don't want to pay for the debug overhead most of the time.

Comment (jdm) [reply gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/Renderer11.cpp]: Bah, I thought I removed that change.

Comment (jdm) [reply gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/Renderer11.cpp]: Fixed.

**Comment by jdm** [comment]
Thanks!

**Pull request merged**